src/navigation/navigation.cc

```diff
@@ -537,14 +572,14 @@ void Navigation::Run() {
 	visualization::ClearVisualizationMsg(local_viz_msg_);
 	visualization::ClearVisualizationMsg(global_viz_msg_);
 
-	// Added this section for anything that we only want to happen once (like Arduino Setup function)
+	// Added this section for anything that we only want to happen once (like setup function)
 	if  (init_){
 		while (init_ and ros::ok()){
 			ros::spinOnce();
 			ros::Rate(10).sleep();
 		}
 		goal_vector_ = Vector2f(4,0);
-		setLocalPlannerWeights(2,1,0);
+		setLocalPlannerWeights(5,2,1);
```

Comment:
This was a nice place to have this for testing, but a better place would be navigation_main.cc since it is a public method

Reply:
That would be good practice, but also our first change to nav_main. Your decision.